python/dgl/nn/pytorch/conv/relgraphconv.py

```diff
@@ -70,7 +70,8 @@ def __init__(self,
                  activation=None,
                  self_loop=False,
                  low_mem=False,
-                 dropout=0.0):
+                 dropout=0.0,
+                 layer_norm=False):
```

Comment:
do you benchmark the benefit of using layer norm in GNN?

Reply:
for mag, 2 or 3% performance gain.